content/runbooks/alertmanager/AlertmanagerFailedToSendAlerts.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Alertmanager Failed To Send Alerts
+weight: 20
+---
+
+# AlertmanagerFailedToSendAlerts
+
+## Meaning
+
+At least one instance is unable to routed alert to the corresponding integration.
+
+## Impact
+
+No impact since another instance will be able to send the notification.
+
+## Diagnosis
+
+Verify that alerts send by each instance have equivalent alert distribution per integration.
```

Comment:
how would you do this, and what do you mean by alert distribution per integration?

Reply:
An integration in alert manager is pager duty, victorops, email, rocket chat etc... basically any external system.

With 2 instances, if one integration fails but not the other for a given integration, comparing them could help to find which one is misbehaving, I believe.

you can check the amount of failed notification for example in promQL `alertmanager_notifications_failed_total{integration="pagerduty"}`

Reply:
ok so basically check that total alerts / integration is the same